### PR TITLE
Create turkish.json

### PR DESCRIPTION
**Pull request by dogagcmnr**
Will continue tomorrow

### CODE REVIEW

File: server/data/words/turkish.json [hunk 1]
```diff
@@ -0,0 +1,4574 @@
...
+  {
+    "word": "suçlamak",
+    "translation": "blame"
+  },
+  {
+    "word": "battaniye",
+    "translation": "blanket"
+  },
+  {
+    "word": kör",
```
Comment (jayehernandez) [inline server/data/words/turkish.json:1067]: Changes look good so far -- it seems like you are missing a quotation mark here which is why Github registers the JSON as invalid. You can use [JSONlint](https://jsonlint.com/) to validate if your JSON is correct.

Let me know if this PR is up for review already :)

**Comment by jayehernandez** [comment]
Hey @dogagcmnr, just checking up on this, is this ready to go? :)

**Comment by dogagcmnr** [comment]
> Hey @dogagcmnr, just checking up on this, is this ready to go? :)

Hey @jayehernandez, i'm going to inform you asap.

**Review by dogagcmnr — COMMENTED** [review]
Reviewed. Please approve it @jayehernandez 👍

**Review by jayehernandez — APPROVED** [review]

**Comment by jayehernandez** [comment]
Hey @dogagcmnr thank you so much for your contribution! The start of the year got quite hectic for me so I wasn't able to merge this earlier. Good work though, I'll be including this in the next release 🥳

**Comment by jayehernandez** [comment]
@all-contributors please add @dogagcmnr for translation

**Pull request merged**